sdk/core/Azure.Core/tests/Data/cert-invalid-data.pem

```diff
@@ -0,0 +1,83 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I would embed this as C# string but don't have a strong opinion.

Reply:
If I can, I will. All this was ported from Identity which, given the public/internal APIs, relies on files.